test/fixtures/clean-snapshot.js

```diff
@@ -33,11 +33,21 @@ const cleanDate = (str) =>
 
 const cleanTime = str => str.replace(/in [0-9]+m?s\s*$/gm, 'in {TIME}')
 
+const cleanZlib = str => str
+    .replace(/shasum:( *).*/g, 'shasum:$1{sha}')
```

Comment:
Is it worth being more precise about the output pattern we expect to see?

```suggestion
    .replace(/shasum:( *)[0-9a-f]{40}/g, 'shasum:$1{sha}')
```

Reply:
Eh it's of questionable value here.  The actual process of consuming and validating those is done in its own suite of tests, this is just a broad output validation.

Reply:
I conflated this w/ integrity, which is harder to regex.  This one is like the `size` one earlier so I'll add it.